.github/workflows/scripts/run-integration-tests-group.sh

```diff
@@ -21,6 +22,11 @@ do
       shift # skip --index
       shift # skip index value
       ;;
+    --tagprefix)
```

Comment:
nit: We only pass test grouping options via args, the rest is passed via env vars.